routes/cohortRoutes.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const router = express.Router();
+
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+const db = knex(knexConfig.development);
+
+const responseStatus = {
+  success: 200,
+  postCreated: 201,
+  badRequest: 400,
+  notFound: 404,
+  serverError: 500
+};
+
+router.get('/', (req, res) => {
+  db('cohorts')
+    .then(cohorts => {
+      res.status(responseStatus.success).json(cohorts);
+    })
+    .catch(err => res.status(responseStatus.serverError).json(err));
+});
+
+router.post('/', async (req, res) => {
+  try {
+    const ids = await db('cohorts').insert(req.body);
+    const cohortResponse = await db('cohorts').where({ id: ids[0] });
+    res.status(responseStatus.postCreated).json(cohortResponse);
+  } catch (error) {
+    if (error.errno === 19) {
+      res.status(responseStatus.badRequest).json({
+        message: 'You must include a name that does not exist in the database.'
+      });
+    } else {
+      res
+        .status(responseStatus.serverError)
+        .json({ message: 'Error adding cohort.' });
+    }
+  }
+});
+
+router.get('/:id', async (req, res) => {
+  try {
+    const animalId = await db('cohorts').where({ id: req.params.id });
+    if (animalId.length === 0) {
```

Comment:
Pretty sure you could re-write as `if (!animalId.length) {` because `0` would evaluate to false, so if the length is equal to `0` it would be equal to false. So pretty sure, `if (!animalId.length) {` would work since that's the same as not equal to 1.